index.bs

```diff
@@ -4608,9 +4585,11 @@ Constructors</h4>
 <dl dfn-type=method dfn-for="AudioBufferSourceNode/constructor">
 	: <dfn>AudioBufferSourceNode(context, options)</dfn>
 	::
-		Let <var>node</var> be a new {{AudioBufferSourceNode}}
-		object. <a href="#audionode-constructor-init">Initialize</a>
-		<var>node</var>, and return <var>node</var>.
+
+		When the constructor is called with a {{BaseAudioContext}} <var>c</var> and
```

Comment:
Nit: since the parameters of the constructor are `context` and `options`, maybe use `<var>context</var>` and `<var>options</var>` instead?

Reply:
Yes, fixed (in the macro).